packages/eslint-plugin/rules/__tests__/jsx-a11y-anchor-has-content.js

```diff
@@ -0,0 +1,278 @@
+/**
+ * External dependencies
+ */
+import { RuleTester } from 'eslint';
```

Comment:
I made the test code a little less verbose since reading through it again made me realize it was hard to understand each test's content